frontend-v2/src/features/data/Data.tsx

```diff
@@ -52,6 +55,17 @@ const Data:FC = () => {
     };
   }
 
+  function downloadCSV() {
+    const csv = generateCSV(dataset, groups, subjectBiomarkers, units);
+    const blob = new Blob([csv], { type: 'text/csv' });
+    const url = window.URL.createObjectURL(blob);
+    const a = document.createElement('a');
+    a.href = url;
+    a.download = 'data.csv';
```

Comment:
This is a placeholder name. Need to set a name (based on the project?) before this merges.